integrations/reporting_channels/gmail/gmail.py

```diff
@@ -50,19 +53,27 @@ def connect(self, raise_errors=True):
         return self.imap
 
 
+    # Add fetch_mailboxes method in the Gmail class
     def fetch_mailboxes(self):
-        response, mailbox_list = self.imap.list()
+        response, data = self.imap.list()
```

Comment:
Any reason to use data as variable name instead of mailbox_list

Reply:
replaced with mailbox_list